docs/README.md

```diff
@@ -1,94 +1,26 @@
 # Introduction
 
-## What is Feast?
+Feast (**Fea**ture **St**ore) is an [open-source](https://github.com/feast-dev/feast) feature store that helps teams define, manage, validate, and serve machine learning features in production at scale. Whether you are training batch or real-time models, Feast reuses your existing data infrastructure to empower both training and inference workflows.
 
-Feast (**Fea**ture **St**ore) is an [open-source](https://github.com/feast-dev/feast) feature store that helps teams 
-operate production ML systems at scale by allowing them to define, manage, validate, and serve features for production 
```

Comment:
Thank you for this!

I don't think we should purely delete all of this content and maybe we can just move it elsewhere?